cmd/crank/xpkg/login.go

```diff
@@ -47,14 +47,37 @@ const (
 )
 
 type loginCmd struct {
+	// Flags. Keep sorted alphabetically.
+	Password string `short:"p" env:"UP_PASSWORD" help:"Password for specified username. '-' to read from stdin."`
+	Token    string `short:"t" env:"UP_TOKEN" xor:"identifier" help:"Token used to authenticate. '-' to read from stdin."`
+	Username string `short:"u" env:"UP_USER" xor:"identifier" help:"Username used to authenticate."`
+
+	// Common Upbound API configuration.
+	// Unfortunately these will show up as a second, alphabetically sorted list
+	// of flags immediately concatenated with the above set.
+	upbound.Flags `embed:""`
+
+	// Internal state. These aren't part of the user-exposed CLI structure.
 	stdin  *os.File
 	client *http.Client
+}
 
-	Username string `short:"u" env:"UP_USER" xor:"identifier" help:"Username used to execute command."`
-	Password string `short:"p" env:"UP_PASSWORD" help:"Password for specified user. '-' to read from stdin."`
-	Token    string `short:"t" env:"UP_TOKEN" xor:"identifier" help:"Token used to execute command. '-' to read from stdin."`
+// Help prints out the help for the login command.
+func (c *loginCmd) Help() string {
+	return `
+Crossplane can be extended using packages. A Crossplane package is sometimes
+called an xpkg. Crossplane supports configuration, provider and function
+packages. 
 
-	upbound.Flags `embed:""`
+A package is an opinionated OCI image that contains everything needed to extend
+Crossplane with new functionality. For example installing a provider package
+extends Crossplane with support for new kinds of managed resource (MR).
```

Comment:
same here about "sometimes".
It's the third copy of this, we should probably move the common part somewhere shared